AnnoyancesFilter/sections/self-promo.txt

```diff
@@ -50,6 +50,8 @@ disk.yandex.*##.mail360-banner
 !-----------------------------------------------
 !
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/103126
+biglobe.ne.jp##.prom
```

Comment:
a lot of these elements are their own ads or another useful links

Reply:
On my end this rule has no hit.

Reply:
@gork7777 please remove this rule